modules/mauth-test-utils/src/main/java/com/mdsol/mauth/test/utils/ProtocolTestSuiteHelper.java

```diff
@@ -0,0 +1,158 @@
+package com.mdsol.mauth.test.utils;
+
+import com.mdsol.mauth.test.utils.model.AuthenticationHeader;
+import com.mdsol.mauth.test.utils.model.SigningConfig;
+import com.mdsol.mauth.test.utils.model.TestCase;
+import com.mdsol.mauth.test.utils.model.UnsignedRequest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProtocolTestSuiteHelper {
+
+  final static String TEST_SUITE_RELATIVE_PATH = "../../mauth-protocol-test-suite/";
+  final static String MWSV2_TEST_CASE_PATH = getFullFilePath("protocols/MWSV2/");
+
+  public final static SigningConfig SIGNING_CONFIG;
+  static {
+    SigningConfig tmp = null;
+    String configFile = getFullFilePath("signing-config.json");
+    try {
+      ObjectMapper objectMapper = new ObjectMapper();
+      byte[] jsonData = Files.readAllBytes(normalizePath(configFile));
+      tmp = objectMapper.readValue(jsonData, SigningConfig.class);
+    } catch (IOException ex) {
+      ex.printStackTrace();
+    }
+    SIGNING_CONFIG = tmp;
+  }
+
+  public final static String PUBLIC_KEY ;
+  static {
+    String tmp = null;
+    String filePath = getFullFilePath("signing-params/rsa-key-pub");
+    try {
+      tmp = new String(Files.readAllBytes(normalizePath(filePath)), Charset.defaultCharset());
```

Comment:
StandardCharsets.UTF_8 please? `defaultCharset` is different across systems and it may not be UTF-8

Reply:
updated as suggested in https://github.com/mdsol/mauth-jvm-clients/pull/133/commits/d0b697b5fe69e9526f046978edf26958c2c292ce